### PR TITLE
FAD-60 selection tool user toggle

### PR DESCRIPTION
**Pull request by sachasmart-weavik**
### Description

> User Toggle Switch
Figma File: https://www.figma.com/file/g7Edk2JCOMLpRAw9v8NJ4c/FA-Product-Wireframe---working-file?node-id=94%3A12407
### Checklist

[FAD-60]

- [x] `CHANGELOG` has been update
ezgif com-gif-maker (1)
d (_if appropriate_)
- [x] Environment variables updated (_if appropriate_)

[FAD-60]: https://weavik.atlassian.net/browse/FAD-60?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by v1n33th — APPROVED** [review]
Awesome and great transitions. Just a minor suggestion with naming

File: packages/client3/components/design-system/toggle/toggle.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import clsx from "clsx";
+
+interface ToggleProps {
+  onChange?: () => void;
+  value: boolean;
+  states: {
+    true: string;
+    false: string;
+  };
```
[the remark this answers is not in the document]
Comment (sachasmart-weavik) [reply packages/client3/components/design-system/toggle/toggle.tsx:9]: How about: 
```
states: {
  selectedState: string;
   state1: string;
   state2:string;
}
```

I'll defer to the group @luansievers. Personally, I like everything nested together.

File: packages/client3/tailwind.config.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ module.exports = {
           accent2: "#7BF8AF",
           accent3: "#D9644A",
           accent4: "#95628A",
+          accent5: "#343E3B",
```
Comment (luansievers) [inline packages/client3/tailwind.config.js:26]: you might need to update your branch with the main

**Review by luansievers — CHANGES_REQUESTED** [review]
update your branch and check the classes again please.
also, add package-lock.json again lol

File: packages/client3/components/design-system/toggle/toggle.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (sachasmart-weavik) [reply packages/client3/components/design-system/toggle/toggle.tsx]: Went with this.

**Comment by sachasmart-weavik** [comment]
I have no idea why the package-lock.json was deleted from `packages/client3`. I have attached a video to show that it is there...


ezgif com-gif-maker-3

File: packages/client3/components/design-system/toggle/toggle.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+let value = true;
+ToggleStory.args = {
+  onChange: () => {
+    value = !value;
+  },
+  states: {
+    selectedState: value,
+    state1: "Backer",
+    state2: "Artist",
```
Comment (luansievers) [inline packages/client3/components/design-system/toggle/toggle.stories.tsx:22]: in my opinion, this is not quite clear what state is active, I mean

`state1, state2, selectedState: true,` what is selected?

myabe change the `selectedState` to `isState1Selected`

**Review by luansievers — CHANGES_REQUESTED** [review]

**Review by luansievers — APPROVED** [review]

**Pull request merged**